R/modify.R

```diff
@@ -174,7 +174,7 @@ imodify <- function(.x, .f, ...) {
 modify_where <- function(.x, .where, .f, ..., .error_call = caller_env()) {
   if (vec_is_list(.x)) {
     out <- vec_proxy(.x)
-    out[.where] <- map(out[.where], .f, ...)
+    out[.where] <- no_zap(map(out[.where], .f, ...), .error_call)
```

Comment:
Was this one of the places where the `.error_call` got prefixed? Just noting that this will need to be updated depending on the order you merge the PRs

Reply:
No, but it probably should've been 😬